modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurator/ClientExtensionProjectConfigurator.java

```diff
@@ -1159,6 +1164,38 @@ private void _validateClientExtension(
 				clientExtension, "membershipType", "open", "private",
 				"restricted");
 		}
+
+		if (Objects.equals(clientExtension.type, "themeCSS")) {
+			String frontendTokenDefinitionFilePath =
+				clientExtension.typeSettings.getOrDefault(
+					"frontendTokenDefinition", ""
+				).toString();
+
+			if (!frontendTokenDefinitionFilePath.isEmpty()) {
+				_validateFrontendTokenDefinitionFile(
+					frontendTokenDefinitionFilePath, project);
+			}
+		}
+	}
+
+	private void _validateFrontendTokenDefinitionFile(
+		String filePath, Project project) {
+
+		try {
+			JsonParser.parseReader(new FileReader(project.file(filePath)));
+		}
+		catch (FileNotFoundException fileNotFoundException) {
+			throw new GradleException(
+				String.format(
+					"File %s not found or it is not a file.",
```

Comment:
From the [wordsmith conventions](https://liferay.atlassian.net/wiki/spaces/ENGR/pages/1511752578/Wordsmith+conventions+in+Liferay#%5BinlineExtension%5D%22Unable-to-find%22%2C-not-%22File-not-found%22), this message should be something like "Unable to find %s"